deployment/src/docker/config.yml

```diff
@@ -0,0 +1,105 @@
+server:
```

Comment:
This seems pretty similar to the file in `./deployment/src/dist/etc/config.yml`, and rather than duplicating it, I wonder if it might be better to use a task like this:

```
task copyConfigTask(type: Copy) {
    from 'src/dist/etc/config.yml'
    into 'src/docker'
}
```

I'm not entirely sure whether you'd need to put this inside the `docker` block or if the `docker` block would need a `dependsOn copyConfigTask` line (or maybe that goes separately)

Reply:
This is a simple fix, as they let you include any path at the root of the docker build context. I will remove the duplicate file.